website/docs/guides/microsoft-fabric-qs.md

```diff
@@ -0,0 +1,314 @@
+
+### Related content
+- [dbt Courses](https://courses.getdbt.com/collections)
+- [About continuous integration jobs](/docs/deploy/continuous-integration)
+- [Deploy jobs](/docs/deploy/deploy-jobs)
+- [Job notifications](/docs/deploy/job-notifications)
+- [Source freshness](/docs/deploy/source-freshness)
+
+## Load data into your Microsoft Fabric warehouse
+
+1. Log in to your [Microsoft Fabric](http://app.fabric.microsoft.com) account.  
+2. On the home page, select the **Synapse Data Warehouse** tile.
+
+    <Lightbox src="/img/quickstarts/dbt-cloud/example-start-fabric.png" width="80%" title="Example of the Synapse Data Warehouse tile" />
+
+3. From **Workspaces** on the left sidebar, navigate to your organization’s workspace. Or, you can create a new workspace to work from; refer to [Create a workspace](https://learn.microsoft.com/en-us/fabric/get-started/create-workspaces) in the Microsoft docs for more details.
+4. Choose your warehouse from the table. Or, you can create a new warehouse; refer to [Create a warehouse](https://learn.microsoft.com/en-us/fabric/data-warehouse/tutorial-create-warehouse) in the Microsoft docs for more details.
+5. Open the SQL editor by selecting **New SQL query** from the top bar. 
+6. Load the Jaffle Shop example data. Copy these statements into the SQL editor:
+
+    ```sql
+    DROP TABLE dbo.customers;
+
+    CREATE TABLE dbo.customers
+    (
+        [ID] [int],
+        [FIRST_NAME] [varchar] (8000),
+        [LAST_NAME] [varchar] (8000)
+    );
+
+    COPY INTO [dbo].[customers]
+    FROM 'https://dbtlabsynapsedatalake.blob.core.windows.net/dbt-quickstart-public/jaffle_shop_customers.parquet'
+    WITH (
+        FILE_TYPE = 'PARQUET'
+    );
+
+    DROP TABLE dbo.orders;
+
+    CREATE TABLE dbo.orders
+    (
+        [ID] [int],
+        [USER_ID] [int],
+        -- [ORDER_DATE] [int],
+        [ORDER_DATE] [date],
+        [STATUS] [varchar] (8000)
+    );
+
+    COPY INTO [dbo].[orders]
+    FROM 'https://dbtlabsynapsedatalake.blob.core.windows.net/dbt-quickstart-public/jaffle_shop_orders.parquet'
+    WITH (
+        FILE_TYPE = 'PARQUET'
+    );
+
+    DROP TABLE dbo.payments;
+
+    CREATE TABLE dbo.payments
+    (
+        [ID] [int],
+        [ORDERID] [int],
+        [PAYMENTMETHOD] [varchar] (8000),
+        [STATUS] [varchar] (8000),
+        [AMOUNT] [int],
+        [CREATED] [date]
+    );
+
+    COPY INTO [dbo].[payments]
+    FROM 'https://dbtlabsynapsedatalake.blob.core.windows.net/dbt-quickstart-public/stripe_payments.parquet'
+    WITH (
+        FILE_TYPE = 'PARQUET'
+    );
+    ```
+
+    <Lightbox src="/img/quickstarts/dbt-cloud/example-load-data-ms-fabric.png" width="80%" title="Example of loading data" />
+
+## Connect dbt Cloud to Microsoft Fabric
+
+1. Create a new project in dbt Cloud. From **Account settings** (using the gear menu in the top right corner), click **+ New Project**.
```

Comment:
To say consistent with other instructions/guides we should probably remove the parenthesis and add it to the beginning of the sentence.